src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Shared.xml

```diff
@@ -0,0 +1,63 @@
+<linker>
```

Comment:
Is .Shared suffix necessary?

Reply:
It technically isn't necessary, but since `ILLink.Descriptors.xml` is the full name of the resource embedded in the assembly, I thought it would be confusing to someone who thought this was the _full_ descriptor file. Putting `Shared` in the file name qualifies it being the "shared" pieces, and not the full file.

Reply:
ok